core/commands/bitswap.go

```diff
@@ -6,17 +6,18 @@ import (
 	"io"
 
 	cmds "github.com/ipfs/go-ipfs/commands"
+	e "github.com/ipfs/go-ipfs/core/commands/e"
```

Comment:
I can just cram it into cmdsutil/cmdkit though. That reminds me, I have to complete the renaming.